maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,76 @@
+
+Download and import idr0021 data
+================================
+
+To download IDR data and re-import elsewhere....
+(See gdoc https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit# )
```

Comment:
The document is not available to external so we should avoid putting this info in a public document